presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.h

```diff
@@ -213,11 +213,13 @@ class VeloxBatchQueryPlanConverter : public VeloxQueryPlanConverterBase {
   VeloxBatchQueryPlanConverter(
       const std::string& shuffleName,
       std::shared_ptr<std::string>&& serializedShuffleWriteInfo,
+      std::shared_ptr<std::string>&& broadcastBasePath,
```

Comment:
Use std::optional<std::string>.

Reply:
presto_protocol generated batchUpdaterequest produces this type definition. Should I create optional out of shared ptr here ?